nessie/jobs/generate_merged_student_feeds.py

```diff
@@ -315,9 +315,8 @@ def generate_merged_enrollment_terms(self, merged_profile, term_id=None):
                 )
                 self.rows['student_enrollment_terms'].append(encoded_tsv_row([sid, term_id, json.dumps(term_feed)]))
             app.logger.debug(
-                f'Enrollment term merge complete (uid={uid}, sid={sid}, term_id={term_id}, ',
-                f'{datetime.now().timestamp() - ts} seconds)',
-            )
```

Comment:
When the closing parenthesis is on its own line, lint-py helpfully suggests that we add trailing commas.